modules/admin_manual/pages/installation/letsencrypt/apache.adoc

```diff
@@ -79,13 +84,18 @@ After that, add an Alias directive for the `/.well-known/acme-challenge` locatio
 == Create an SSL VirtualHost Configuration
 
 We recommend creating a separate file for storing the `SSL` directives.
-This is because, when the certificate has been created, you can use this file in any SSL-enabled VirtualHost configuration for which the certificate is valid, without reissuing the SSL certificate.
+If these directives already exist in this Virtual Host,
+delete them and include the file instead.
+This is because, when the certificate has been created, you can use this file in any 
+SSL-enabled VirtualHost configuration for which the certificate is valid, without reissuing
+the SSL certificate.
 
-....
+[source,console]
+----
 cd /etc/apache2/
 sudo mkdir ssl_rules
 touch ssl_rules/ssl_mydom.tld
-....
+----
 
 ./etc/apache2/ssl_rules/ssl_mydom.tld
 [source,apacheconf]
```

Comment:
Since the file is being improved, why not change this to `apache` instead of `apacheconf` as well?